src/components/flat-table/flat-table-row/flat-table-row.style.js

```diff
@@ -6,6 +6,45 @@ import StyledFlatTableCheckbox from "../flat-table-checkbox/flat-table-checkbox.
 import StyledFlatTableHeader from "../flat-table-header/flat-table-header.style";
 import StyledIcon from "../../icon/icon.style";
 
+const stickyColumnFocusStyling = (index, theme) => {
+  return `
+    border-bottom: 1px solid transparent;
+    ${index === 0 ? `border-left: 1px solid ${theme.colors.focus};` : ""}
+    background-clip: padding-box;
+    z-index: ${theme.zIndex.overlay};
```

Comment:
Why does it need a z-index?

Reply:
Honestly not sure, it was existing code that I moved as I needed to define it for any other columns that were before the sticky header. I'll have a little dig around and see if it's needed

Reply:
@dawidzarzycki  it looks like you added the z-index here, do you know why it was done? Whilst I can't see what impact removing them will have I'm a little hesitant to do it in case it effects something I'm not aware of

Reply:
If I'm not wrong I just changed hardcoded zIndex into one that comes from theme object

Reply:
@sjohnson-sage looks like it's needed to ensure the focus outline isn't hidden behind the sticky columns

![image](https://user-images.githubusercontent.com/44157880/112805716-dee02a80-906d-11eb-86af-a6da2024142c.png)

Reply:
looks like we need it for the row header focus overlay